_pkgdown.yml

```diff
@@ -0,0 +1,49 @@
+url: https://lintr.r-lib.org
+template:
+  bootstrap: 5
+
+  includes:
+    in_header: |
+      <script defer data-domain="lintr.r-lib.org,all.tidyverse.org" src="https://plausible.io/js/plausible.js"></script>
+
+reference:
+- title: Linting
+  contents:
+  - lint_dir
+  - lint_file
+  - lint_package
+  - linters_with_defaults
+  - linters_with_tags
+  - read_settings
+  - exclude
+  - modify_defaults
+  - checkstyle_output
+
+- title: Individual linters
+  contents:
+  - all_undesirable_functions
+  - ends_with("linter")
+
+- title: Linter bunldes
```

Comment:
These are tag sections. Only `default_linters` actually contains a list of linters. The others are just help pages linking to linters with a similar purpose.

"Linters by tag" maybe?

Reply:
Hmmmm, is there a reason you don't just use `@family` to cross-link these sections together? 

I'm not sure that the groups are that useful unless you want to add more information to the title (i.e. what are configurable linters? what does it mean to be a readability linter?)

Reply:
Each tag has a short documentation like you mentioned.
See `R/linter_tag_docs.R`.

Reply:
But you don't see that until you click into the specific tag.